api/BUILD

```diff
@@ -245,7 +245,6 @@ proto_library(
         "//envoy/service/discovery/v3:pkg",
         "//envoy/service/endpoint/v3:pkg",
         "//envoy/service/event_reporting/v3:pkg",
-        "//envoy/service/extension/v3:pkg",
```

Comment:
@kyessenov is this the issue you were seeing? Did you have some resolution?

Reply:
Yeap. No, I just manually edited it every time.